docs/_tutorials/getting-started.md

```diff
@@ -26,10 +28,10 @@ model_engine, optimizer, _, _ = deepspeed.initialize(args=cmd_args,
 
 `deepspeed.initialize` ensures that all of the necessary setup required for
 distributed data parallel or mixed precision training are done
-appropriately under the hood.  In addition to wrapping the model, DeepSpeed can
+appropriately under the hood. In addition to wrapping the model, DeepSpeed can
 construct and manage the training optimizer, data loader, and the learning rate
 scheduler based on the parameters passed to `deepspeed.initialize` and the
-DeepSpeed [configuration file](#deepspeed-configuration).
+DeepSpeed [configuration file](#deepspeed-configuration). Note that DeepSpeed automatically updates the learning rate at every training step.
```

Comment:
It is more precise that DeepSpeed executes the learning rate scheduler at every training step. The distinction is that a given lr_scheduler.step() may or may not change  the learning rate depending on the scheduler algorithm.

Reply:
That's right.  Made the change. Thanks.